lib/internal/primordials.js

```diff
@@ -0,0 +1,446 @@
+
+    'use strict';
```

Comment:
remove this file, we don't want primordials.

Reply:
Unfortunately the Node core uses primordials everywhere.
I've replaced with a custom slimmed down primordials code.